cli/tests/test_completion.rs

```diff
@@ -381,6 +381,7 @@ fn test_revisions() {
     test_env.jj_cmd_ok(&repo_path, &["b", "c", "immutable_bookmark"]);
     test_env.jj_cmd_ok(&repo_path, &["commit", "-m", "immutable"]);
     test_env.add_config(r#"revset-aliases."immutable_heads()" = "immutable_bookmark""#);
+    test_env.add_config(r#"revset-aliases."siblings" = "@-+ ~@""#);
```

Comment:
nit: can you add test for alias definition containing newline char?

Reply:
Done. Looks like clap automatically cuts off the help text after one line.